src/openshift/storage.ts

```diff
@@ -21,11 +20,16 @@ export class Storage extends OpenShiftItem {
 
         if (!storageName) return null;
 
-        const mountPath = await vscode.window.showInputBox({prompt: "Specify the mount path", validateInput: (value: string) => {
-            if (validator.isEmpty(value.trim())) {
-                return 'Invalid mount path';
+        const mountPath = await vscode.window.showInputBox({
+            prompt: "Specify the mount path",
+            validateInput: (value: string) => {
+                let validationMessage = Storage.emptyName('Empty mount path', value.trim());
+                if (!validationMessage) validationMessage = Storage.isAbsolute('Invalid mount path', value.trim());
```

Comment:
On the subject of unix paths - path.isAbsolute returns true for windows style paths when called on windows. That would allow windows users to pass validation without actually using a unix path.

Reply:
Path should be always valid absolute unix path.

Reply:
use  isPosixAbsolute here:
```
Storage.isPosixAbsolute('Invalid mount path',  value.trim());
```